src/index.tsx

```diff
@@ -3,13 +3,11 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import Home from './components/Home/Home';
+import LayoutComponent from './components/Layout/LayoutComponent';
 
-function App() {
-  return (
-    <>
-      <Home />
-    </>
-  );
-}
-
+const App: React.FC = () => (
+  <LayoutComponent>
```

Comment:
Rename the component to MainLayout

```suggestion
  <MainLayout>
```

Reply:
Done